script.js

```diff
@@ -1,4 +1,254 @@
+/**
+ * scissors-paper-stone
+ */
+
+//The program internally randomly chooses 'scissors', 'paper', or 'stone'.
+//define function for generating a random integer
+var generateRandomInteger = function (max) {
+  return 1;
```

Comment:
I think this was leftover from your checking? It would cause your function to only return 1
Otherwise good use of helper functions